api/time_bucket.md

```diff
@@ -1,58 +1,59 @@
-## time_bucket() 
+# time_bucket()
+The `time_bucket` function is based on the standard PostgreSQL `date_trunc`
+function. It allows for arbitrary time intervals instead of the second, minute,
+and hour intervals provided by `date_trunc`. The return value is the bucket's
+start time.
 
-This is a more powerful version of the standard PostgreSQL `date_trunc` function.
-It allows for arbitrary time intervals instead of the second, minute, hour, etc.
-provided by `date_trunc`. The return value is the bucket's start time.
-Below is necessary information for using it effectively.
-
-<highlight type="tip">
-TIMESTAMPTZ arguments are bucketed by the time at UTC. So the alignment of 
+`TIMESTAMPTZ` arguments are bucketed by the time in UTC, so the alignment of
 buckets is on UTC time. One consequence of this is that daily buckets are
-aligned to midnight UTC, not local time.
-
-If the user wants buckets aligned by local time, the TIMESTAMPTZ input should be
-cast to TIMESTAMP (such a cast converts the value to local time) before being
-passed to time_bucket (see example below).  Note that along daylight savings
-time boundaries the amount of data aggregated into a bucket after such a cast is
-irregular: for example if the bucket_width is 2 hours, the number of UTC hours
-bucketed by local time on daylight savings time boundaries can be either 3 hours
-or 1 hour.
+aligned to midnight UTC, not local time. If you want buckets aligned by local
+time, cast the `TIMESTAMPTZ` input to `TIMESTAMP`, which converts the value to
+local time, before you pass it to `time_bucket`. For an example, see the sample
+use in this section.
+
+Note that daylight savings time boundaries means that the amount of data
+aggregated into a bucket after such a cast can be irregular. For example, if the
+`bucket_width` is 2 hours, the number of UTC hours bucketed by local time on
+daylight savings time boundaries can be either three hours or one hour.
+
+<highlight type="important">
+Month, year, and timezone arguments are not supported by the `time_bucket`
```

Comment:
This is slightly misleading and contradicts our regular terminology. Months and/or years can be specified in the **same** argument of the function - bucket width. In other words, there are no separate arguments for months and years. Better just omit "argument" word in this context.